src/Core/Services/Implementations/LicensingService.cs

```diff
@@ -124,6 +127,8 @@ private async Task DisableOrganizationAsync(Organization org, ILicense license,
             org.ExpirationDate = license?.Expires ?? DateTime.UtcNow;
             org.RevisionDate = DateTime.UtcNow;
             await _organizationRepository.ReplaceAsync(org);
+
+            await _mailService.SendLicenseExpiredAsync(new List<string> {org.BillingEmail}, true);
```

Comment:
@kspearrin , on styling for should this have spaces between the curly braces and inner item(s)?
i.e. - `{ org.BillingEmail }` vs. `{org.BillingEmail}`

Reply:
yes, add spaces